packages/input-tel-dropdown/src/LionInputTelDropdown.js

```diff
@@ -247,6 +247,16 @@ export class LionInputTelDropdown extends LionInputTel {
     this.__syncRegionWithDropdown = this.__syncRegionWithDropdown.bind(this);
   }
 
+  /**
+   * @param {import('lit-element').PropertyValues } changedProperties
+   */
+  firstUpdated(changedProperties) {
+    super.firstUpdated(changedProperties);
+    if (changedProperties.has('activeRegion')) {
```

Comment:
This could still fail if phoneUtil is lazy loaded on a slow internet connection.